AssemblerLib/MeshComponentMap.cpp

```diff
@@ -57,7 +58,7 @@ MeshComponentMap::MeshComponentMap(
     std::size_t comp_id = 0;
     _num_global_dof = 0;
     _num_local_dof = 0;
-    for (auto const c : components)
+    for (auto const& c : components)
     {
         assert(c != nullptr);
```

Comment:
Since `` const c`` has been changed to `` const&``, is ``assert(c != nullptr);`` still good?